Python/ceval.c

```diff
@@ -2495,7 +2498,7 @@ PyEval_GetFuncDesc(PyObject *func)
 #define C_TRACE(x, call) \
```

Comment:
For clarity and debugging, should maybe have an explicit assert here that `current_frame` is an interpreter frame (`f_executable` is a code object), since the code below assumes it.